metrics/storage/webtooling.sh

```diff
@@ -47,10 +53,61 @@ help(){
 cat << EOF
 Usage: $0 <count>
    Description:
-        <count> : Number of containers to run.
+       <count> : Number of containers to run.
 EOF
 }
 
+clean() {
+	local containers=( $(sudo ctr c list -q) )
+	local i=""
+	for i in "${containers[@]}"; do
+            	info "loop $count of $max: sleeping for 1 second"
+		sudo ctr task kill -s SIGKILL "$i"
+		sudo ctr c rm "$i"
+	done
+}
+
+verify_task_is_completed_on_all_containers() {
+	local containers=( $(sudo ctr c list -q) )
+	local sleep_secs=10
+	local max=$(bc <<<"$timeout / $sleep_secs")
+	local wip_list=()
+	local count=1
+	local sum=1
```

Comment:
@GabyCT The issue you mention is here, the sum variable need to be initialized to 0.